src/DatePicker/DurationUtilities.elm

```diff
@@ -418,6 +419,16 @@ dayPickedOrBetween zone day hovered ( startSelectionTuple, endSelectionTuple ) =
             ( day == startPickerDay || day == endPickerDay, isDayBetweenDates day startPickerDay endPickerDay )
 
 
+isPickedDaySelectionTuple : PickerDay -> Maybe ( PickerDay, Posix ) -> Bool
+isPickedDaySelectionTuple day selectionTuple =
+    case selectionTuple of
+        Just ( pickerDay, _ ) ->
+            pickerDay == day
+
+        _ ->
+            False
+
+
```

Comment:
Could you add a test for this?

Reply:
I tried 😄 not really a testing specialist... maybe you'd like to check again?
https://github.com/mercurymedia/elm-datetime-picker/pull/38/files#diff-d777ea602f139bf04890b5279b0d5222024e4f9b48a16cbe82fa24d4e2e6743f

Reply:
Looks good to me :)